mmdet/ops/nms/nms_wrapper.py

```diff
@@ -93,7 +96,7 @@ def nms(boxes, scores, iou_threshold, score_thr=0.0, max_num=-1, offset=0):
         inds = order.masked_select(select)
     else:
         if max_num < 0:
-            max_num = sys.maxsize
+            max_num = boxes.size()[0]
```

Comment:
Isn't it equivalent to `boxes.size(0)` or `boxes.shape[0]`?

Reply:
Yes, I will replace that with `box.shape[0]`.